packages/discovery-react-components/src/components/SearchFacets/utils/searchFacetInterfaces.ts

```diff
@@ -33,3 +41,25 @@ export interface CollectionItem {
   id: string;
   label: string;
 }
+
+export interface QueryAggregationWithName extends DiscoveryV2.QueryAggregation {
+  field?: string;
+  count?: number;
+  name?: string;
+}
+
+export interface SelectableFieldFacetWithCategory {
+  key?: string;
```

Comment:
shouldn't `key` be required?

Reply:
I was matching it to `SelectableDynamicFacets` since that is the similar selectable facet interface also used in `CollapsibleFacetsGroup`, but in this case, it is always required, so updating to required here 👍